metaflow/plugins/aws/batch/batch_cli.py

```diff
@@ -150,8 +150,6 @@ def kill(ctx, run_id, user, my_runs):
 @click.option("--tmpfs-tempdir", is_flag=True, help="tmpfs requirement for AWS Batch.")
 @click.option("--tmpfs-size", help="tmpfs requirement for AWS Batch.")
 @click.option("--tmpfs-path", help="tmpfs requirement for AWS Batch.")
-# TODO: Maybe remove it altogether since it's not used here
-@click.option("--ubf-context", default=None, type=click.Choice([None, "ubf_control"]))
```

Comment:
removing this completely leads to `@parallel` breaking on batch. Keep the original fix instead and change comment to

```
# NOTE: ubf-context is not explicitly used, but @parallel decorator tries to pass this so keep it for now.
```